recipe/fix-msvc.diff

```diff
@@ -0,0 +1,13 @@
+--- c++/src/kj/CMakeLists.txt.bak   2024-05-20 14:48:26.618071654 -0500
++++ c++/src/kj/CMakeLists.txt       2024-05-20 14:53:15.106427359 -0500
+@@ -139,6 +139,9 @@
+ )
+ if(NOT CAPNP_LITE)
+   add_library(kj-async ${kj-async_sources})
++  if (MSVC)
++    set_property(SOURCE async.c++ APPEND PROPERTY COMPILE_OPTIONS "/O1")
++  endif()
```

Comment:
Would be nice to know why this was necessary (perhaps a comment next to `patches:` in meta.yaml`) - compiler bug with default optimization level?

Reply:
Done in 0e16a59bc6e1ac917730bbe729ec0aa78b1a04ad